homeassistant/components/honeywell/climate.py

```diff
@@ -295,7 +295,10 @@ def _set_temperature(self, **kwargs) -> None:
             # Get current mode
             mode = self._device.system_mode
             # Set hold if this is not the case
-            if getattr(self._device, f"hold_{mode}") is False:
+            if (
```

Comment:
This will avoid the stack trace but it won't actually set the `emheat` hold mode. Is that ok?

Reply:
Yes.  As far as i know, emheat is a unique situation that doesn't have a hold mode.

Reply:
`emheat` doesn't currently exist, but we can add it. We would also need to update `somecomfort` first though.